shared/src/main/scala/uk/gov/homeoffice/drt/arrivals/ArrivalsDiff.scala

```diff
@@ -82,7 +81,16 @@ case class ArrivalsDiff(toUpdate: SortedMap[UniqueArrival, Arrival], toRemove: I
       case (acc, (key, arrival)) =>
         acc.get(key) match {
           case Some(fws) =>
-            acc + (key -> fws.copy(apiFlight = arrival, lastUpdated = Option(nowMillis)))
+            val (feedSources, paxSources) = fws.splits.foldLeft((arrival.FeedSources, arrival.PassengerSources)) {
+              case ((accFs, accPs), split) if Set(ApiSplitsWithHistoricalEGateAndFTPercentages, Historical).contains(split.source) =>
+                val totalPax = Option(split.totalPax)
+                val transPax = if (split.transPax > 0) Option(split.transPax) else None
+                val fs = if (split.source == ApiSplitsWithHistoricalEGateAndFTPercentages) ApiFeedSource else HistoricApiFeedSource
```

Comment:
with Historic word in ApiSplitsWithHistoricalEGateAndFTPercentages , fs should be HistoricApiFeedSource rather than ApiFeedSource ?

Reply:
Actually, the reason the word historic appears in thie source name is because it uses historic egate and fast track percentages. It's the source name we use for live API splits